### ISSUE TITLE
Is there any image augmentation operation during the training ? some early-stopping metrics to avoid the network being overfitting ?

### ISSUE THREAD

**Issue by a6225301** [fizyr/keras-maskrcnn#29]
Hello, Is there any image augmentation operation during the training ? 
Meanwhile, any some early-stopping metrics to avoid the network being overfitting ?
Thanks~

**Comment by hgaiser** [comment]
By default, during training, only horizontal flipping is enabled : https://github.com/fizyr/keras-maskrcnn/blob/master/keras_maskrcnn/bin/train.py#L145, but more [options](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/utils/transform.py#L202-L214) are available.

I've never really experienced much overfitting with retinanet / maskrcnn. Are you experiencing overfitting? The problem with Keras and the early stopping callback is that it depends on a metric. I haven't yet managed to change the mAP computations into a Keras metric. Normally I keep an eye on the mAP during training and select the snapshot with the highest mAP values.
